metron-interface/metron-rest/src/main/java/org/apache/metron/rest/controller/HdfsController.java

```diff
@@ -0,0 +1,89 @@
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.metron.rest.controller;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.apache.hadoop.fs.Path;
+import org.apache.metron.rest.RestException;
+import org.apache.metron.rest.service.HdfsService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+@RestController
+@RequestMapping("/api/v1/hdfs")
+public class HdfsController {
+
+  @Autowired
+  private HdfsService hdfsService;
+
+  @ApiOperation(value = "Reads a file from HDFS and returns the contents")
+  @ApiResponse(message = "Returns file contents", code = 200)
+  @RequestMapping(value = "/list", method = RequestMethod.GET)
+  ResponseEntity<List<String>> list(@ApiParam(name = "path", value = "Path to HDFS directory", required = true) @RequestParam String path) throws RestException {
+    return new ResponseEntity<>(hdfsService.list(new Path(path)), HttpStatus.OK);
+  }
+
+  @ApiOperation(value = "Reads a file from HDFS and returns the contents")
+  @ApiResponse(message = "Returns file contents", code = 200)
+  @RequestMapping(method = RequestMethod.GET)
+  ResponseEntity<String> read(@ApiParam(name = "path", value = "Path to HDFS file", required = true) @RequestParam String path) throws RestException {
+    String contents = hdfsService.read(new Path(path));
+    if (contents != null) {
+      return new ResponseEntity<>(hdfsService.read(new Path(path)), HttpStatus.OK);
+    } else {
+      return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+    }
+
+  }
+
+  @ApiOperation(value = "Writes contents to an HDFS file")
+  @ApiResponse(message = "Contents were written", code = 200)
+  @RequestMapping(method = RequestMethod.POST)
+  ResponseEntity<Void> write(@ApiParam(name="path", value="Path to HDFS file", required=true) @RequestParam String path,
```

Comment:
This is potentially a dangerous operation. Can you add to the description that this operation will overwrite the contents of a file that already exists?